commands/command_ls_files.go

```diff
@@ -38,7 +39,25 @@ func lsFilesCommand(cmd *cobra.Command, args []string) {
 			return
 		}
 
-		Print("%s %s %s", p.Oid[0:showOidLen], lsFilesMarker(p), p.Name)
+		if debug {
+			Print(
+				"  filepath: %s\n"+
+					"      size: %d\n"+
+					"  checkout: %v\n"+
+					"downloaded: %v\n"+
+					"       oid: %s\n"+
+					"   oidtype: %s\n"+
```

Comment:
I'd probably smash the oid into one line:

    oid: %s %s
    oid: sha256 abcdef...

Reply:
Agreed 👍